lib/projectImport.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
 
             //format and put each found group address into the object tree
             _.each(groupAddresses, function (groupAddress) {
-                const gaName = groupAddress.getAttribute("Name");
+                const gaName = groupAddress.getAttribute("Name").replace(tools.FORBIDDEN_CHARS, '_');
                 let autoread = true;
                 let dptObj = {};
                 let fullPath = tools.formatGa(gaName);
```

Comment:
assuming mittelgruppe, hauptgruppe .. can also have this problem, not only ga names

it needs a unified rule for gaName and tools.formatGa

Reply:
moved filtering into formatting